packages/core/src/__stampers__/api/mobile/stamper.ts

```
@@ -75,4 +75,28 @@ export class ReactNativeKeychainStamper implements ApiKeyStamperBase {
     const { stampHeaderName, stampHeaderValue } = await stamper.stamp(payload);
     return { stampHeaderName, stampHeaderValue };
   }
+
+  async sign(
+    payload: string,
+    publicKeyHex: string,
+    format: SignatureFormat = SignatureFormat.Der,
+  ): Promise<string> {
+    const privateKey = await this.getPrivateKey(publicKeyHex);
+    if (!privateKey) {
+      throw new Error(`No private key found for public key: ${publicKeyHex}`);
+    }
+    const stamper = new ApiKeyStamper({
+      apiPublicKey: publicKeyHex,
+      apiPrivateKey: privateKey,
+    });
+
+    switch (format) {
+      case SignatureFormat.Raw: {
+        return stamper.sign(payload, SignatureFormat.Raw);
+      }
+      case SignatureFormat.Der:
```

let’s error in the default case. That way, if we add a new signature type and forget to update this, it will fail loudly instead of creating an incorrect signature